src/engine/state.rs

```diff
@@ -234,9 +234,9 @@ mod tests {
         let assets = AssetManager::new();
         let pipe = Pipeline::new();
         let pool = Arc::new(ThreadPool::new(2));
-        let planner = Planner::from_pool(World::new(), pool.clone());
+        let planner: Planner<()> = Planner::from_pool(World::new(), pool.clone());
 
-        let mut engine = Engine { manager: assets, pipe: pipe, planner: planner, pool: pool };
+        let mut engine = unsafe { ::std::mem::uninitialized() };
```

Comment:
@vitvakatu Yes, if you want to keep the test. It's because travis cannot create a window, which is required by `GfxDevice`, which is required by `Engine`.